distributed/diagnostics/nvml.py

```diff
@@ -1,17 +1,26 @@
 import os
 
+import dask
+
 try:
     import pynvml
 except ImportError:
     pynvml = None
 
+nvmlEnabled = True
 nvmlInitialized = False
 nvmlLibraryNotFound = False
 nvmlOwnerPID = None
 
 
 def init_once():
-    global nvmlInitialized, nvmlLibraryNotFound, nvmlOwnerPID
+    global nvmlEnabled, nvmlInitialized, nvmlLibraryNotFound, nvmlOwnerPID
+
+    nvmlEnabled = dask.config.get("distributed.diagnostics.nvml")
```

Comment:
IIUC this `nvmlEnabled` variable appears to be equivalent to the the `distributed.diagnostics.nvml` config value. If that's the case, I'd prefer to not introduce a new variable and instead just use the config value

Reply:
This is a good point, addressed in https://github.com/dask/distributed/pull/4893/commits/fed9e9770c89b0b0909130cb607942cef3db0d60 .